lib/renderer.coffee

```diff
@@ -25,6 +25,10 @@ exports.toHtml = (text='', filePath) ->
 exports.toRawHtml = (text='', filePath) ->
   render text, filePath
 
+exports.getBlockId = (bufferRowPosition) ->
+  if exports.blocksPositions
```

Comment:
you use `@blocksPosition` but in CoffeeScript (and ES2015) you must bing `@`


```coffeescript
render = (text='', filePath) =>
  return Promise.resolve() unless atom.config.get('asciidoc-preview.defaultAttributes')?

  new Promise (resolve, reject) =>
    attributes = makeAttributes()
    options = makeOptions filePath

    taskPath = require.resolve('./worker')
    task = Task.once taskPath, text, attributes, options

    task.on 'asciidoctor-load:success', ({blocksPositions}) =>
      console.log "before success@blocksPositions", @blocksPositions
      @blocksPositions = blocksPositions
      console.log "after success@blocksPositions", @blocksPositions
...
```

See the fat arrows I have add.